00-HTML-CSS-basics/html 1.1.4/html1.1.4.html

```diff
@@ -0,0 +1,32 @@
+<html>
+	<head>
+		<title>Expensas</title>
+		<meta charset="utf-8">
+		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
+		<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+	</head>
+	<header><strong>Expensas</strong></header>
+	<body>
+		<div>
+			<table width="400px">
```

Comment:
You shouldn't set style properties for tables from html, do it from css with a class for your tables, rows and cells.